tools/deployment/package-helm/values.yaml

```diff
@@ -18,25 +18,101 @@ image:
     pullPolicy: "Always"
     tag: "main"
 
+workerConcurrency: 8
+
 clpConfig:
   package:
     storage_engine: "clp-s"
     query_engine: "clp-s"
 
+  # Location (e.g., directory) containing any logs you wish to compress. Must be reachable by all
+  # workers.
+  logs_input:
+    type: "fs"
+
+    # NOTE: This directory will be exposed inside the container, so symbolic links to files outside
+    # this directory will be ignored.
+    directory: "/"
+
   database:
     type: "mariadb"  # "mariadb" or "mysql"
     port: 30306
     name: "clp-db"
 
+  compression_scheduler:
+    jobs_poll_delay: 0.1  # seconds
+    logging_level: "INFO"
+    max_concurrent_tasks_per_job: 0  # A value of 0 disables the limit
+
+  query_scheduler:
+    port: 7000
```

Comment:
Note this value is unused. Since query_scheduler is not a user interfacing service, we should remove this configuration field to avoid confusion. @coderabbitai submit an issue